astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -690,22 +690,29 @@ def test_aa_high_precision():
 
     The absence of polar motion within Skyfield and the disagreement between Skyfield and Horizons
     make high precision comparisons to those codes difficult.
+
+    Updated 2020-11-29, after the comparison between codes became even better,
+    down to 100 nas (reflecting consistency in approach at least).
```

Comment:
@StuartLittlefair - your comment is why I added this to the docstring. I think it is OK to tie ourselves this tightly, so that if an error starts happening, we are forced to look into what changed. Maybe I should make that more explicit for our future selves?

Reply:
I think so, what I'm hoping to avoid is future us implementing a new precession model from the IAU (for example) and wondering why this test fails and what we should do about it.

Reply:
How about if I change the comment to read as follows (passing by you first to avoid re-running CI needlessly - `[ci skip]` no longer works...)
```
    NOTE: the agreement reflects consistency in approach between two codes,
    not necessarily absolute precision.  If this test starts failing, the
    tolerance can and should be weakened *if* it is clear that the change is
    due to an improvement (e.g., a new IAU precession model).
```

Reply:
Perfect!